modules/service/keeper/keeper_test.go

```
@@ -350,13 +349,14 @@ func (suite *KeeperTestSuite) TestRegisterCallback() {
 
 func (suite *KeeperTestSuite) TestKeeperRequestContext() {
 	consumer := testConsumer
+
 	providers := []sdk.AccAddress{testProvider}
 
 	suite.setServiceDefinition()
 
 	blockHeight := int64(1000)
 	ctx := suite.ctx.WithBlockHeight(blockHeight)
-	suite.app.BeginBlocker(ctx, abci.RequestBeginBlock{})
+	suite.app.BeginBlocker(ctx)
```

golangci-lint

Error return value of `suite.app.BeginBlocker` is not checked

(errcheck)